.github/workflows/release-please.yml

```diff
@@ -0,0 +1,31 @@
+# .github/workflows/release-please.yml
```

Comment:
this file as well should be deleted https://github.com/microsoftgraph/msgraph-sdk-go/blob/main/.github/workflows/git-release.yml